tests/features/multisite_registration/multisite_registration_communities.feature

```diff
@@ -0,0 +1,175 @@
+@api @javascript

```

Comment:
Since this feature refers to the og (or communities) version of the platform, you should add the tag @communities